### ISSUE TITLE
Update for more recent versions of Python?

### ISSUE THREAD

**Issue by HughMacdonald** [sam-mosleh/signalr-async#9]
Hi,

Is there a plan to update this for more recent versions of Python? It appears to only officially support up to 3.10

When using it with Python 3.12, I'm seeing a failure here:

https://github.com/sam-mosleh/signalr-async/blob/master/signalr_async/connection.py#L125

> AttributeError: 'WebSocketWriter' object has no attribute 'send'

**Comment by HughMacdonald** [comment]
I think that the short-term fix here would be to restrict the aiohttp dependency to 3.10.* only (it fails with 3.11.*)

**Comment by sam-mosleh** [comment]
I'm planning to use structured concurrency and ditch aiohttp entirely

**Comment by HughMacdonald** [comment]
Fair enough. For now, in my project, I'm just restricting the aiohttp version at the higher level for now.
